include/nn/ac/ac_cpp.h

```diff
@@ -24,10 +25,24 @@ namespace ac
  * An ID number representing a network configuration. These are the same IDs as
  * shown in System Settings' Connection List.
  */
-typedef uint32_t ConfigIdNum;
+using ConfigIdNum = uint32_t;
 
 /**
- * C++ linkage for the autoconnect API, see \link nn::ac \endlink for use.
+ * The configuration for a given network profile.
+ * \sa NetConfCfg
+ */
+using Config = NetConfCfg;
+
+using ErrorCode = uint32_t;
+
+enum Status {
+    STATUS_FAILED     = -1,
+    STATUS_OK         =  0,
+    STATUS_PROCESSING =  1,
+};
+
+/**
+* C++ linkage for the autoconnect API, see \link nn::ac \endlink for use.
  * Cafe provides mangled symbols for C++ APIs, so nn::ac is actually static
  * inline calls to the mangled symbols under nn::ac::detail.
  */
```

Comment:
Instead of doing this, the more recently added headers now simply use [asm labels](https://gcc.gnu.org/onlinedocs/gcc-4.4.0/gcc/Asm-Labels.html) after the function declaration to change the name to be used during linkage. See for example the functions in nn_pdm: https://github.com/devkitPro/wut/blob/ad730b9dcb7f7489e47a6f910d9f610708ac1eb5/include/nn/pdm/pdm_cpp.h#L92-L94
It might be a good idea to make the change for nn_ac in this PR.

Reply:
Yes, that looks better. I'll change to that then.